### PR TITLE
create docs

### PR DESCRIPTION
**Pull request by banma12956**

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'PARL'
+copyright = '2019, nlp-ol@baidu.com'
+author = 'nlp-ol@baidu.com'
+
+# The full version, including alpha/beta/rc tags
+release = '1.1'
```
Comment (hanszeng) [inline source/conf.py]: Should use global version information, like parl.__version__ or reference: https://github.com/PaddlePaddle/PARL/blob/255ef4f7877195d9f57a3cb728323562457b34b6/setup.py#L59

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+html_logo = './PARL-logo-2.png'
+
+import sys
+import os
+sys.path.insert(0, os.path.abspath('../'))
+import parl
+release = '1.1'
```
Comment (zenghsh3) [inline docs/conf.py]: Should use global version information, like parl.\_\_version\_\_ or reference: https://github.com/PaddlePaddle/PARL/blob/255ef4f7877195d9f57a3cb728323562457b34b6/setup.py#L59

File: source/conf.py (hunk 1 above)
Comment (zenghsh3) [inline source/conf.py]: Should use global version information, like parl.\_\_version\_\_ or reference: https://github.com/PaddlePaddle/PARL/blob/255ef4f7877195d9f57a3cb728323562457b34b6/setup.py#L59

File: docs/conf.py (hunk 2 above)
Comment (TomorrowIsAnOtherDay) [reply docs/conf.py]: release = parl.\_\_version\_\_

File: docs/hello.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+hello world!
```
Comment (TomorrowIsAnOtherDay) [inline docs/hello.rst]: Documents for testing locally should be removed.

File: docs/make.bat [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (TomorrowIsAnOtherDay) [inline docs/make.bat]: This file is for building in a windows environment. We don't need it.

File: source/index.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
+.. PARL_docs documentation master file, created by
```
Comment (TomorrowIsAnOtherDay) [inline source/index.rst]: Why this file is similar to the docs/index.rst ?

**Review by TomorrowIsAnOtherDay — CHANGES_REQUESTED** [review]

**Comment by TomorrowIsAnOtherDay** [comment]
We are going to merge this PR.
Subsequent job:
1. check whether the docs could be used for readdocs.io.
2. add unit test code in teacmcity.

**Review by TomorrowIsAnOtherDay — APPROVED** [review]

**Pull request merged**